api-tests/apply_route_test.go

```diff
@@ -467,6 +475,7 @@ func TestApplyPsmdbReturnedVersions(t *testing.T) {
 		{"3.6-latest", "1.7.0", nil, "3.6.21-10.0"},
 		{"3.6-latest", "1.6.0", nil, "3.6.21-10.0"},
 		{"3.6-latest", "1.5.0", nil, "3.6.19-7.0"},
+		{"6.0-recommended", "1.14.0", nil, "6.0.3-2"},
 		{"5.0-recommended", "1.13.0", nil, "5.0.11-10"},
```

Comment:
for 1.14.0 we should have recommended for all versions: 
{"4.4-recommended", "1.13.0", nil, "4.4.16-16"},
{"5.0-recommended", "1.13.0", nil, "5.0.11-10"},

Reply:
I'm not sure, if you think you can update the PR directly, it will be faster I think.